ngi_reports/stockholm/project_summary.py

```diff
@@ -27,27 +27,18 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         super(Report, self).__init__(config, LOG, working_dir, **kwargs)
 
         ## Project name - collect from the command line if we have it
-        if kwargs.get('project') is not None:
-            self.project_info['ngi_name'] = kwargs['project']
-            self.project_name = self.project_info['ngi_name']
-        else:
-            ## Check to see if we have the project ID yet
-            try:
-                self.project_name = self.project_info['ngi_name']
-            except KeyError:
-                self.LOG.error("No project name found - please specify using '--project'")
-                raise KeyError("No project name found - please specify using '--project'")
+        self.project_name = kwargs.get('project', self.project_info.get('ngi_name'))
+        if not self.project_name:
+            self.LOG.error("No project name/id provided - please specify using '--project'")
+            raise SystemExit
```

Comment:
Used `SystemExit` as we know whats the error was i.e. not providing a `project` name (It is logged).

```
2016-02-29 10:32:34,333 - NGI Reports - INFO - Report type: project_summary
2016-02-29 10:32:34,534 - NGI Reports - INFO - Found 0 Piper setup XML files..
2016-02-29 10:32:34,534 - NGI Reports - ERROR - No project name/id provided - please specify using '--project'
```

 I wont mind raising an `AttributeError`. It just gives some extra lines to the console with line numbers where it failed, which would be useful for debugging though not in this case.  

```
2016-02-29 10:23:41,982 - NGI Reports - INFO - Report type: project_summary
2016-02-29 10:23:42,347 - NGI Reports - INFO - Found 0 Piper setup XML files..
2016-02-29 10:23:42,347 - NGI Reports - ERROR - No project name/id provided - please specify using '--project'
Traceback (most recent call last):
  File "/Users/senthilkumarpanneerselvam/anaconda/envs/report/bin/ngi_reports", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/Users/senthilkumarpanneerselvam/opt/ngi_reports/scripts/ngi_reports", line 185, in <module>
    make_reports(**kwargs)
  File "/Users/senthilkumarpanneerselvam/opt/ngi_reports/scripts/ngi_reports", line 41, in make_reports
    report = report_mod.Report(config, LOG, working_dir, **kwargs)
  File "/Users/senthilkumarpanneerselvam/opt/ngi_reports/ngi_reports/stockholm/project_summary.py", line 33, in __init__
    raise AttributeError
AttributeError
```